3741/3/Readme.md

```diff
@@ -0,0 +1,19 @@
+**This simple web app of place ranking**
+
+!Prepare for lunch:
+- delete next string in main_controller.rb
```

Comment:
Cool you are recommending to remove the lines instead of doing it by yourself

Reply:
fix that issues, no need more delete strings